package.json

```diff
@@ -54,5 +54,9 @@
         12
       ]
     }
-  }
+  },
+  "files": [
```

Comment:
I'm not sure I agree with a whitelist approach being dangerous, but I'm happy to use `npmignore` if you'd prefer that.

Tests should not be published to npm, however. They are used for developers to ensure that the module works correctly. There is no need for a user of the library to have to download the test files if they want to use the library.

Anyone working on the module who needs the tests can get them from the Git repo.